src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderCard.tsx

```diff
@@ -74,6 +78,35 @@ export const LimitOrderCard: FC<LimitOrderCardProps> = ({
 
   const buyAsset = useSelectorWithArgs(selectAssetById, buyAssetId)
   const sellAsset = useSelectorWithArgs(selectAssetById, sellAssetId)
+  const explorerTxLink = buyAsset?.explorerTxLink
+
+  const { data: tradeTxLink } = useQuery({
+    queryKey: ['cowTrade', uid],
+    queryFn:
+      status === OrderStatus.FULFILLED && explorerTxLink
+        ? () => {
+            const chainId = fromAccountId(accountId).chainId
+            const network = assertGetCowNetwork(chainId)
+            return axios.get<Trade[]>(
+              `${getConfig().VITE_COWSWAP_BASE_URL}/${network}/api/v1/trades?orderUid=${uid}`,
+            )
+          }
+        : skipToken,
+    select: data => {
+      const txHash = data?.data?.[0]?.txHash
+      return `${explorerTxLink}/${txHash}`
+    },
+  })
+
+  const maybeExternalLink = useMemo(() => {
```

Comment:
This could do with a loading skeleton, there is a bit of a jarring `Tag` resize on first load as we don't have the `tradeTxLink` straight away.